Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:20-alpine3.18 as base
+FROM node:20-alpine3.16 as base
 
-ENV HTTP_PROXY='http://proxy.local:8080'
-ENV HTTPS_PROXY='http://proxy.local:8080'
+ARG HTTP_PROXY=''
+ARG HTTPS_PROXY=''
```

Comment:
Presuming you can't do aware with these fields given that you're passing in `--builg-args` in the CLI?

Reply:
I thought we needed them for the docker build but seemingly nor, so I have removed.